src/app/shared/navbar/navbar.component.html

```diff
@@ -1,150 +1,63 @@
+              <div aria-hidden="true" class="m-auto h-0.5 w-6 rounded bg-white transition duration-300"></div>
+              <div aria-hidden="true" class="m-auto mt-2 h-0.5 w-6 rounded bg-white transition duration-300"></div>
+            </label>
+
+            <div
+              class="peer-checked:translate-x-0 fixed inset-0 w-[calc(100%-4.5rem)] translate-x-[-100%]  border-r shadow-xl transition duration-300 lg:border-r-0 lg:w-auto lg:static lg:shadow-none lg:translate-x-0">
+              <div class="flex flex-col h-full justify-between lg:items-center lg:flex-row bg-[#141d2c]">
+                <ul class="px-6 pt-32 text-gray-700 space-y-8 md:px-12 lg:space-y-0 lg:flex lg:space-x-12 lg:pt-0">
+                  <li>
+                    <a routerLink="/"
+                      class="group relative before:absolute before:inset-x-0 before:bottom-0 before:h-5 before:origin-right before:scale-x-0 before:bg-[#8599b6] before:rounded-md before:transition before:duration-200 hover:before:origin-left hover:before:scale-x-100">
+                      <span class="relative   text-white font-semibold">Home</span>
+                    </a>
+                  </li>
+                  <li>
+                    <a
+                      class="group relative before:absolute before:inset-x-0 before:bottom-0 before:h-5 before:origin-right before:scale-x-0 before:bg-[#8599b6] before:transition before:rounded-md before:duration-200 hover:before:origin-left hover:before:scale-x-100">
+                      <span class="relative  text-white font-semibold">PQRS</span>
+                    </a>
+                  </li>
+                  <li>
+                    <a routerLink="/mission"
+                      class="group relative before:absolute before:inset-x-0 before:bottom-0 before:h-5 before:origin-right before:scale-x-0 before:bg-[#8599b6] before:transition before:rounded before:duration-200 hover:before:origin-left hover:before:scale-x-100">
+                      <span class="relative  text-white font-semibold">Mission</span>
+                    </a>
+                  </li>
+                  <li>
+                    <a routerLink="/vision"
+                      class="group relative before:absolute before:inset-x-0 before:bottom-0 before:h-5 before:origin-right before:scale-x-0 before:bg-[#8599b6] before:transition before:rounded before:duration-200 hover:before:origin-left hover:before:scale-x-100">
+                      <span class="relative  text-white font-semibold">Vission</span>
+                    </a>
+                  </li>
+                </ul>
+
+                <div class="border-t py-8 px-6 md:px-12 md:py-16 lg:border-t-0 lg:border-l lg:py-0 lg:pr-0 lg:pl-6">
+                  <a href="#"
+                    class="block px-6 py-3 rounded-full bg-gradient-to-r from-[#8599b6] to-[#E7F6F2] text-center text-[#141d2c] font-semibold">
+                    Sign up
```

Comment:
No deberia decir Sign In? Y desde el sign In ya por medio de un enlace en el formulario redirigir a al Sign Up